Mathlib/Data/Finset/Card.lean

```diff
@@ -443,6 +443,16 @@ theorem exists_ne_map_eq_of_card_lt_of_maps_to (hc : #t < #s) {f : α → β}
   contrapose
   exact hz x hx y hy
 
+/-- a special case of `exists_ne_map_eq_of_card_lt_of_maps_to` where `t` is `s.image f` -/
+theorem exists_ne_map_eq_of_card_image_lt [DecidableEq β] {f : α → β} (hc : #(s.image f) < #s) :
+    ∃ x ∈ s, ∃ y ∈ s, x ≠ y ∧ f x = f y :=
+  exists_ne_map_eq_of_card_lt_of_maps_to hc (coe_image (β := β) ▸ Set.mapsTo_image f s)
+
+/-- a variant of `exists_ne_map_eq_of_card_image_lt` using `InjOn` -/
```

Comment:
If you are referring to a declaration in a docstring, you have to fully qualify it for it to properly link-ify in the generated docs website (AFAIK). Especially since this referenced lemma exists in both the Set and Finset namespaces

Reply:
Sorry I somehow missed this comment. Fixed.